.github/workflows/CI.yml

```diff
@@ -17,14 +17,19 @@ defaults:
 jobs:
   test:
 
-    name: ${{ matrix.cfg.os }}, 🐍=${{ matrix.python-version }}, program=${{ matrix.cfg.conda-env }}
+    name: ${{ matrix.cfg.os }}, 🐍=${{ matrix.python-version }}, program=${{ matrix.cfg.conda-env }}, pydantic=${{ matrix.pydantic-version }}
     runs-on: ${{ matrix.cfg.os }}
 
     strategy:
       fail-fast: false
       matrix:
         python-version:
           - "3.9"
+          - "3.10"
```

Comment:
(blocking) This is spawning 18 CI runs, which is a lot. Please bring this under 15 before merging. I recommend dropping py3.10, since it's unlikely that something with 3.10 would fail while the 3.9 and 3.11 tests pass.